addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3920,6 +3920,18 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
         }
         return this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    onBuilt() {
+        // Flip classes should no longer be used but are still present in some
+        // theme snippets.
+        if (this.$target[0].querySelector('.o_we_flip_x, .o_we_flip_y')) {
```

Comment:
In master, we will be able to remove the class uses from all snippets? To be sure I understand: it is only about adding the right background-image src with a flip parameter? I wonder if we should keep your onBuilt auto-fix even in master or not.

Reply:
>  it is only about adding the right background-image src with a flip parameter?

Yes, but we also need to include `background-position: 50% 0%;` for flipping the shape vertically. In the master, we will be able to remove the classes if we modify the background-image style if the 'o_we_shape' <div> in all the templates.